mkdocs/docs/HPC/teaching_training.md

```diff
@@ -0,0 +1,139 @@
+  - teachers are automatically made moderators of these dedicated groups
+  - moderators can populate these groups with vsc-ids of group members in the VSC accountpage
+  - only these vsc-ids will then be able to access a group[abc] folder, and will have read/write access
+
+If you need any of these additional folders, do indicate under Optional storage requirements:
+/shared YES-NO
+/subgroups HOW MANY SUBGROUPS
+
+The data policy for the dedicated course storage is the following: on the indicated end date of your course, the course directory will be made unaccessible to all. One year after the end date it will be permanently removed. We assume that teachers/moderators always have an own copy of the course data as a starting point for a next course.
+
+
+## Resource requirements beyond the interactive cluster
+
+We assume that your course requirements are such that the [interactive cluster {https://docs.hpc.ugent.be/interactive_debug/} ] can be used. If these resources are insufficient, you will need to request and motivate a reservation.
+
+Indicate which cluster you would need and the number of nodes, cores, GPUs.
+Also clearly indicate when you would need these resources, i.e. the dates and times of each course session.
+
+Be aware that students will have no access to the reservation outside the course sessions.
+This might be relevant when requesting a custom application.
+
+Reservations take away precious resources for all HPC users, so only request this when it is really needed for your course. In our experience, the [interactive cluster {https://docs.hpc.ugent.be/interactive_debug/} ] is more than sufficient for the majority of cases.
+
+
+## Specific software
+
+In case you need software for your course/workshop that is unavailable or that needs to be updated, make a separate [software installation request {https://www.ugent.be/hpc/en/support/software-installation-request } ]. Mail hpc@ugent.be the OTRS ticket number in your request for a course/workshop.
+
+We will try to make the software available before the start of your course/workshop, but this is best effort, depending on the load of the support team and the complexity of your software request. Typically, software installation requests must be made at least one month before the course/workshop starts.
+
+Ideally, courses/workshops rely on software that is already in use (and thus also well tested).
+
+
+## Custom Interactive Application in the webportal
+
+HPC-UGent can create a custom interactive application in the web portal for your course/workshop. Typically, this is a generic interactive application such as cluster desktop, Jupyter notebook, ? in which a number of options are preset or locked down: e.g. the number of cores, software version, cluster selection, autostart code, etc. This could make it easier for teachers and students, since students are less prone to making mistakes and don?t have to spend time copy-pasting specific settings. 
```

Comment:
```suggestion
HPC-UGent can create a custom interactive application in the web portal for your course/workshop. Typically, this is a generic interactive application such as cluster desktop, Jupyter notebook, ? in which a number of options are preset or locked down: e.g. the number of cores, software version, cluster selection, autostart code, etc. This could make it easier for teachers and students, since students are less prone to making mistakes and do not have to spend time copy-pasting specific settings. 
```